Minor edit to getting_started.md

aloysiousBenoy opened the pull request:
Correction for minor error spotted.
"Let's visit the endpoint endpoint we saved earlier."  here the **repetition of the word "endpoint"** seemed to be a mistake made while writing the document.

abdelhai commented on docs/libjs/run.md:
````diff
@@ -0,0 +1,75 @@
+
+```ruby
+prog_name
+```
````
omit these lines

abdelhai commented on docs/libjs/run.md:
````diff
@@ -0,0 +1,75 @@
+**Response**
+
+```javascript
+'Wilkommen in Berlin!';
````
no semicolon needed. the "javascript" is only for syntax highlighting

abdelhai commented on docs/libjs/run.md:
````diff
@@ -0,0 +1,75 @@
+**Command (Teletype Dash)**
+
+```ruby
+prog_name kreuzberg
````
omit

abdelhai commented on docs/resources_js/files.md:
````diff
@@ -0,0 +1,99 @@
+const markdown = buf.toString('utf8');
+```
+
+See the Node.js documentation to read your data from a **`Buffer`**
````
could you link to it?

abdelhai commented on js_web_frameworks.md:
```diff
@@ -0,0 +1,57 @@
+
+#### 1) Install the Framework
+
+Open a DETA program and [install the framework dependency](/packages/) using `???` in the [Studio Teletype](/teletype/).
```
```suggestion
Open a DETA program and [install the framework dependency](/packages/) using `npm install package` in the [Studio Teletype](/teletype/).
```

carbaa commented on docs/js_web_frameworks.md:
````diff
@@ -0,0 +1,57 @@
+```javascript
+const detalib = require('detalib');
+const express = require('express');
+const app = detalib.App(express());
````
That wouldn't work because App is inside detalib variable.

carbaa commented on docs/libjs/cron.md:
````diff
@@ -0,0 +1,17 @@
+
+```javascript
+const detalib = require('detalib');
+const app = detalib.App();
````
going with const {app} = require('detalib') since there's an 'app' object now in detalib.

carbaa commented on docs/libjs/http.md:
```diff
@@ -0,0 +1,19 @@
+The DETA JS library delegates http routing to external frameworks. Consider using [an external framework in your DETA program](/web_frameworks/) such as [Express.js](https://expressjs.com/) or [Koa](https://koajs.com/).
+
+Import the web framework and wrap the external web framework using `detalib.App()`. Make calls to the web framework as you would outside of DETA.
```
i made my edit on it. let me know if you like it.

aloysiousBenoy committed:
Minor edit to getting_started.md

Correction for minor error spotted.
"Let's visit the endpoint endpoint we saved earlier."  here the repetition of the word "endpoint" seemed to be a mistake made while writing the document.
